templates/base.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>{% block title %}Taxi Service{% endblock %}</title>
```

Comment:
Correct style for HTML files according to guidelines
https://github.com/mate-academy/py-task-guideline/tree/main/html_settings